test/test.ts

```diff
@@ -1621,6 +1621,48 @@ test('is.all', t => {
 	});
 });
 
+test('is.formData', t => {
+	const data = new window.FormData();
+	t.true(is.formData(data));
+	t.false(is.formData({}));
+	t.false(is.formData(undefined));
+	t.false(is.formData(null));
+
+	t.notThrows(() => {
+		assert.formData(data);
+	});
+	t.throws(() => {
+		assert.formData({});
+	});
+	t.throws(() => {
+		assert.formData(undefined);
+	});
+	t.throws(() => {
+		assert.formData(null);
+	});
+});
+
+test('is.urlSearchParams', t => {
+	const params = new window.URLSearchParams();
```

Comment:
```suggestion
	const searchParameters = new URLSearchParams();
```

`URLSearchParams` is a global in Node.js